package.json

```diff
@@ -1,25 +1,28 @@
 {
   "name": "uport-credentials",
-  "version": "1.1.7",
+  "version": "1.2.0-alpha.0",
   "description": "Library for interacting with uport profiles and attestations",
   "main": "lib/index.js",
+  "umd:main": "lib/index.umd.js",
+  "modules": "lib/index.mjs",
+  "source": "src/index.ts",
+  "types": "lib/index.d.ts",
   "files": [
     "lib",
     "dist",
     "src",
+    "esm",
     "tutorial"
   ],
   "scripts": {
     "test": "./node_modules/.bin/jest",
     "test:ci": "./node_modules/.bin/jest -u --coverage",
-    "build:es5": "./node_modules/.bin/babel src -d lib",
+    "build:js": "./node_modules/.bin/microbundle",
     "build:browser": "./node_modules/.bin/webpack --config webpack.config.js",
-    "build": "npm run build:es5 && npm run test:ci && npm run build:browser",
-    "build:docs:html": "./node_modules/.bin/jsdoc src/* README.md -d docs",
-    "build:docs:md": "./node_modules/jsdoc-to-markdown/bin/cli.js -f src/* --template \"./template.hbs\" -c js2doc.conf > docs/reference/index.md; echo",
-    "build:docs": "npm run build:docs:html && npm run build:docs:md",
+    "build": "npm run build:js && npm run test:ci && npm run build:browser",
+    "build:docs": "echo 'PLEASE UPDATE REFEREN DOCS MANUALLY'",
```

Comment:
Typo here in "REFERENCE."  Why does this need to be done manually instead of using a typedoc command?